### DISCUSSION TITLE
Tokyo_night_night support?

### DISCUSSION THREAD

**Discussion by DrTeagle** [nix-community/stylix#597] [Q&A]
Howdy,
Currently the only real thing preventing me from fully integrating this into my setup is the lack of tokyo night support, by that I mean tokyo_night_night for tmux and kitty. is this a limitation of base16?

**Comment by DrTeagle** [comment]
curently the stylix look is this through tokyo-night-dark:
![image](https://github.com/user-attachments/assets/55a5498d-8478-43a6-b17d-6283b5ae7179)

and the tokyo_night_night is this:
![image](https://github.com/user-attachments/assets/a75a7199-4c6d-49c9-a167-64b669ce9fb3)

**Comment by DrTeagle** [comment]
Ive tried all other dark configs and they all fail to reproduce the tokyo_night_night color scheme

**Comment by DrTeagle** [comment]
seems like it might be a my end issue

**Reply by trueNAHO** [reply]
So the issue has resolved itself? In that case, feel free to close this discussion.

**Reply by DrTeagle** [reply]
whoops yeah sorry it is not resolved on my end, I would appreciate help, but it is not specifically an issue with this repo,so I will close this discussion.

**Reply by trueNAHO** [reply]
> whoops yeah sorry it is not resolved on my end, I would appreciate help

AFAIK, all colorschemes should be functionally equivalent. The only possible problem is that the colorscheme values are not "beautifully" applied. In the future, https://github.com/danth/stylix/issues/534 resolves such inconsistencies.

**Reply by DrTeagle** [reply]
however I have others unable to reproduce this issue with tokyo_night so it seems to be a my end issue, I would appreciate help(or at least guidance on how to diagnose the issue myself):
https://github.com/DockterTeagle/myNixOS

**Reply by trueNAHO** [reply]
> however I have others unable to reproduce this issue with tokyo_night so it seems to be a my end issue, I would appreciate help(or at least guidance on how to diagnose the issue myself): https://github.com/DockterTeagle/myNixOS

Have you `reboot`ed? :eyes:

**Reply by DrTeagle** [reply]
yes, many times it seems that it is only a problem in the home-manager setup, though I could be wrong, as the only setup that is systemwide is bootloader and login screen.

**Reply by DrTeagle** [reply]
still having trouble with this would like help? trying out foot

**Reply by DrTeagle** [reply]
I checked the repo and tokyonight_night is not the same as tokyonight_dark

**Reply by trueNAHO** [reply]
Could you re-explain what exactly the problem is? Does tokyo_night_night look wrong, while tokyo-night-dark looks correct?

In that case, the base16 limitation might cause such subtle differences to look incorrect:

> - [ ] **(4)** Support base24 scheme [7] and Vim highlight groups [5] to resolve styling inconsistencies
>
> [5]: https://github.com/danth/stylix/issues/249
> [7]: https://github.com/danth/stylix/issues/252
>
> -- [NAHO, "Roadmap"](https://github.com/danth/stylix/issues/534#issue-2490009394)
